vpr/src/place/place_delay_model.cpp

```diff
@@ -18,10 +20,7 @@
 #    include "serdes_utils.h"
 #endif /* VTR_ENABLE_CAPNPROTO */
 
```

Comment:
Please add a comment at the top of the file saying what kind of routines should go in here and what the purpose of the file is.

Reply:
Update: Now I see you have one in the header file.  Might be enough, although I tend to think it's good to put it in the .cpp too.